flexmeasures/data/models/planning/linear_optimization.py

```diff
@@ -339,9 +339,16 @@ def cost_function(m):
     model.costs = Objective(rule=cost_function, sense=minimize)
 
     # Solve
-    results = SolverFactory(current_app.config.get("FLEXMEASURES_LP_SOLVER")).solve(
-        model
-    )
+    solver_name = current_app.config.get("FLEXMEASURES_LP_SOLVER")
+    opt = SolverFactory(solver_name)
+
+    if "highs" in solver_name.lower():
+        try:
+            results = opt.solve(model)
+        except RuntimeError:
+            results = opt.solve(model, load_solutions=False)
```

Comment:
Maybe we can mention [your pyomo PR](https://github.com/Pyomo/pyomo/issues/2920) here.

And what is the effect of `load_solutions=False` ?

Reply:
Good idea!

Reply:
I can't quite follow - do we not need the solutions? If not, why go through a try-except block in the first place? If yes, what functionality of ours is affected by not having them loaded?